CONTRIBUTING.md

```diff
@@ -10,3 +10,301 @@ https://qiskit.org/documentation/contributing_to_qiskit.html
+stderr, and log messages and add them as attachments to the tests run so
+output can be associated with the test case it originated from. However, if
+you run tests with `stestr` outside of these mechanisms by default the streams
+are not captured. To enable stream capture just set the
+`QISKIT_TEST_CAPTURE_STREAMS` env variable to `1`. If this environment
+variable is set outside of running with `stestr` the streams (STDOUT, STDERR,
+and logging) will still be captured but **not** displayed in the test runners
+output. If you are using the stdlib unittest runner a similar result can be
+accomplished by using the
+[`--buffer`](https://docs.python.org/3/library/unittest.html#command-line-options)
+option (e.g. `python -m unittest discover --buffer ./test/python`).
+
+### Code style
+
+The Arraylias repo uses black for code formatting and style and pylint
+is run to do a deeper check on the code. You can run these checks locally with
+
+```
+tox -elint
+```
+
+If there is a code formatting issue identified by black you can just run
+``black`` locally to fix this (or ``tox -eblack`` which will install it and
+run it).
+
+
+### Development Cycle
+
+The development cycle for Arraylias is all handled in the open using
+the project boards in Github for project management. We use milestones
+in Github to track work for specific releases. The features or other changes
+that we want to include in a release will be tagged and discussed in Github.
+As we're preparing a new release we'll document what has changed since the
+previous version in the release notes.
+
+### Branches
+
+* `main`:
+
+The master branch is used for development of the next version of Arraylias.
+It will be updated frequently and should not be considered stable. The API
+can and will change on master as we introduce and refine new features.
+
+* `stable/*` branches:
+Branches under `stable/*` are used to maintain released versions of Arraylias.
+It contains the version of the code corresponding to the latest release for
+that minor version on pypi. For example, stable/0.8 contains the code for the
+0.8.2 release on pypi. The API on these branches are stable and the only changes
+merged to it are bugfixes.
+
+### Release cycle
+
+When it is time to release a new minor version of Arraylias we will:
+
+1.  Create a new tag with the version number and push it to github
+2.  Change the `master` version to the next release version.
+
+The release automation processes will be triggered by the new tag and perform
+the following steps:
+
+1.  Create a stable branch for the new minor version from the release tag
+    on the `master` branch
+2.  Build and upload binary wheels to pypi
+3.  Create a github release page with a generated changelog
+4.  Generate a PR on the meta-repository to bump the Arraylias version and
+    meta-package version.
+
+The `stable/*` branches should only receive changes in the form of bug
+fixes.
```

Comment:
I only copied these sections from qiskit-dynamics. Please confirm if you want to change this procedure and cycle.

Reply:
Yeah I figure there's no change to this. I think the only question is if we want to just keep the link to Terra and leave the rest of this blank, or copy it all over. I know we copy it over in Dynamics but could make a fresh decision here.

Reply:
Certainly, it is good time to decide this issue. IIn my opinion, the use of arraylias is not limited to qiskit and it is good to copy it from Dynamics or terra. Rather than, it might make sense to change Dynamics one?

Reply:
Sorry. As long as this repo exists under qiskit-extension, it might be better obey the qiskit instruction and enough to make the link to qiskit.